src/Classes/GemSelectControl.lua

```diff
@@ -187,7 +189,7 @@ function GemSelectClass:UpdateSortCache()
 	GlobalCache.useFullDPS = dpsField == "FullDPS"
 	local calcFunc, calcBase = self.skillsTab.build.calcsTab:GetMiscCalculator(self.build)
 	-- Check for nil because some fields may not be populated, default to 0
-	local baseDPS = (calcBase.Minion and calcBase.Minion.CombinedDPS) or (calcBase[dpsField] ~= nil and calcBase[dpsField]) or 0
+	local baseDPS = (dpsField == "FullDPS" and calcBase[dpsField] ~= nil and calcBase[dpsField]) or (calcBase.Minion and calcBase.Minion.CombinedDPS) or (calcBase[dpsField] ~= nil and calcBase[dpsField]) or 0
```

Comment:
This logic is redundant.  If the first part of the logic is true, so is the second part

```lua
(dpsField == "FullDPS" and calcBase[dpsField] ~= nil and calcBase[dpsField]) or (calcBase[dpsField] ~= nil and calcBase[dpsField])
```

Reply:
Right, but the `calcBase.Minion` is in between. It's an order of operation thing. If "FullDPS" is specified as the "sorting" field I want it to use it over Minions as FullDPS accounts for minions internally in it's roll-up already. If the sorting field is not FullDPS then I'm keeping the original behavior.
Remember in `A or B or C` if A is evaluated as True then Lua doesn't even evaluate B or C.

Reply:
I got focused on the logic, not that it was assigning it to a variable, carry on, then.